### PR TITLE
License change: CeCILL B/C -> MIT

### PR DESCRIPTION
**Pull request by strub**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Adrien Koutsos
+Alley Stoughton
+Asif Mallik
+Benedikt Schmidt
+Benjamin Gregoire
+César Kunz
+Christian Doczkal
+Cécile Baritel-Ruet
+Davide Ramaglietta
+Davy Guillaume
```
Comment (fdupress) [inline AUTHORS:12]: Guillaume Davy

**Review by fdupress — APPROVED** [review]
Good from my POV.

Should we be more careful about `ecField` and `ecRing`?

**Comment by strub** [comment]
> Should we be more careful about `ecField` and `ecRing`?

The files have their own copyright header.

**Comment by bgregoir** [comment]
I approve it ...

**Pull request merged**